File: benchmark/corpus/08_memory/c_split_rows.py
# Splits the same line into fields again and again and keeps every copy.
line = ",".join(str(k) for k in range(1, 41))
rows = []
while True:
    rows.append([int(x) * 2 for x in line.split(",")])  # DANGER
print(len(rows))

File: benchmark/corpus/08_memory/c_split_rows.js
// Splits the same line into fields again and again and keeps every copy.
const line = Array.from({ length: 40 }, (_, k) => String(k + 1)).join(",");
const rows = [];
while (true) {
  rows.push(line.split(",").map((x) => Number(x) * 2)); // DANGER
}
console.log(rows.length);
